share/completions/connmanctl.fish

```diff
@@ -0,0 +1,40 @@
+function __fish_print_connman_services
+    connmanctl services | string replace -r '.* (.*)' '$1'
+end
+
+function __fish_print_connman_technologies
+    connmanctl technologies | string match '*Type*' | string replace -r '  Type = (.*)' '$1'
+end
+
+function __fish_print_connman_vpnconnections
+    connmanctl vpnconnections | string replace -r '.* (.*)' '$1'
+end
+
+complete -f -c connmanctl -n "test (count (commandline -opc)) -lt 2" -a "state" -d "Shows if the system is online or offline"
```

Comment:
In that case, the simple thing is okay.

It might be nice to add a comment about it, though.